### PR TITLE
Barkha_Assignment_24_45mins

### PR DESCRIPTION
**Pull request by BarkhaPJun20**
Validate given phone number.
+91 9765463742 --> valid
09765463742 --> valid
976 546 3742 --> valid
9765463742 --> valid
9765463A42 --> Invalid
976546342 --> Invalid
6976546342 --> Invalid
image

### CODE REVIEW

File: src/barkhaPJun20/VerifyMobileNumber.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package barkhaPJun20;
+
+public class VerifyMobileNumber {
+	
+	void validMobileNumber(String num) {
+		//Exa.+91 9765463742
+		if (num.length()==14) {
```
Comment (SwapnaWJun20) [inline src/barkhaPJun20/VerifyMobileNumber.java:14]: Here, you can write both if conditions together by using && operator.

File: src/barkhaPJun20/VerifyMobileNumber.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		}else 
+		System.out.println("Number is Invalid");
+	}	
+	//out of Desire Length
+	if (num.length()>14  || num.length()<10)
+		System.out.println("Number is Invalid");
+	}	
+	public static void main(String[] args) {
+		VerifyMobileNumber  mobileNumber=new VerifyMobileNumber ();
+		String mobileNum="9765463A42";
```
Comment (SwapnaWJun20) [inline src/barkhaPJun20/VerifyMobileNumber.java]: Mob no can be directly added while calling method.

File: src/barkhaPJun20/VerifyMobileNumber.java (hunk 1 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/VerifyMobileNumber.java:14]: Changes Done

File: src/barkhaPJun20/VerifyMobileNumber.java (hunk 2 above)
Comment (BarkhaPJun20) [reply src/barkhaPJun20/VerifyMobileNumber.java]: Changes Done

File: src/barkhaPJun20/VerifyMobileNumber.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		//out of Desire Length
+		else 
+			System.out.println("Number is Invalid");
+		}	
+public static void main(String[] args) {
+		VerifyMobileNumber  mobileNumber=new VerifyMobileNumber ();
+		//String mobileNum="+91 9765463742";
+		//System.out.println("Mobile Number provided to validate is: "+mobileNum);
+		mobileNumber.validMobileNumber("+91 8565463742");
+	}
```
Comment (SwapnaWJun20) [inline src/barkhaPJun20/VerifyMobileNumber.java:52]: Please try to check 979 567894 this no as input, this is not giving proper output. As it has space in between and the length is 10.

Comment (BarkhaPJun20) [reply src/barkhaPJun20/VerifyMobileNumber.java:52]: Changes Done

**Review by SwapnaWJun20 — APPROVED** [review]

**Comment by SwapnaWJun20** [comment]
Also for future reference, you can use single if loop with && and || operators.

**Pull request merged**